packages/react-components/tests/task-summary/task-summary-accordion.test.tsx

```diff
@@ -0,0 +1,129 @@
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { TaskSummaryAccordion, TaskSummaryAccordionInfo } from '../../lib';
+import { getActorFromStatus, sortTasksByState } from '../../lib/task-summary/task-summary-utils';
+
+const taskSummary = {
+  'af8faee9-84ca-41ea-8bb6-8493cc9f824c': {
+    end_time: { sec: 0, nanosec: 0 },
+    start_time: { sec: 0, nanosec: 0 },
+    state: 1,
+    status:
+      'Moving [tinyRobot/tinyRobot1]: ( 9.81228 -6.98942 -3.12904) -> ( 6.26403 -3.51569  1.16864) | Remaining phases: 1 | Remaining phases: 6',
+    submission_time: { sec: 0, nanosec: 0 },
+    task_id: 'af8faee9-84ca-41ea-8bb6-8493cc9f824c',
+  },
+  'am8faee9-84ca-41ea-8bb6-8493cc9f8249': {
+    end_time: { sec: 0, nanosec: 0 },
+    start_time: { sec: 0, nanosec: 0 },
+    state: 0,
+    status:
+      'Moving [tinyRobot/tinyRobot2]: ( 9.81228 -6.98942 -3.12904) -> ( 6.26403 -3.51569  1.16864) | Remaining phases: 1 | Remaining phases: 6',
+    submission_time: { sec: 0, nanosec: 0 },
+    task_id: 'am8faee9-84ca-41ea-8bb6-8493cc9f8249',
+  },
+};
+
+describe('Renders correctly', () => {
+  let task: RomiCore.TaskSummary;
+  beforeEach(() => {
+    task = Object.assign({}, Object.values(taskSummary)[0]);
```

Comment:
This can "taint" the test data if a test modifies an object inside it, e.g.

```ts
it('bad test', () => {
  task.start_time.sec = 10;
});

it('other test', () => {
  expect(task.start_time.sec).toBe(0); // fails
});
```

I recommend to use a function to create a new copy of task for each test.

Reply:
yeah, that's a better solution, I was assigning another reference on each test. I created a getTaskObject(). Thanks!